dnfdaemon-client/utils.cpp

```diff
@@ -21,10 +21,39 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include <unistd.h>
 
+#include <iostream>
+
 namespace dnfdaemon::client {
 
 bool am_i_root() noexcept {
     return geteuid() == 0;
 }
 
+bool userconfirm(Context & ctx) {
```

Comment:
In dnf4 we have base.output.userconfirm() that is also used in plugins. Additionaly it is capable to handle translated choices. I also belive that microdnf will need such a functionality. What about to move it to libdnf-cli

Reply:
Please can you add `TODO` about planned movement to share the code. Anyway it does not support of translated symbols therefore there is a room for future improvement.